frontend/tests/acceptance/pages/addNewJobApplication.js

```diff
@@ -0,0 +1,63 @@
+const { I } = inject();
+
+module.exports = {
+    fields: {
+        title: '//label[contains(text(),"Title")]/parent::div//input[contains(@class,"MuiInputBase-input")]',
+        company: '//label[contains(text(),"Company")]/parent::div//input[contains(@class,"MuiInputBase-input")]',
+        priority: '//label[contains(text(),"Priority")]/parent::div//div[contains(@class,"MuiSelect-select")]',
+        status: '//label[contains(.,"Status")]/parent::div//div[contains(@class,"MuiSelect-select")]',
```

Comment:
these all xpath seems same except their value in `label contains text`. You can be more DRY with the use of something like: 
```suggestion
        addJobApplicationFieldsXpath: '//label[contains(text(),"%s")]/parent::div//input[contains(@class,"MuiInputBase-input")]',
```
and use `util.format(addJobApplicationFieldsXpath, "Title")` where you need the path!